src/tools/illink/test/Mono.Linker.Tests.Cases/Inheritance.Interfaces/DefaultInterfaceMethods/StaticDefaultInterfaceMethodOnStruct.cs

```diff
@@ -0,0 +1,45 @@
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+
+namespace Mono.Linker.Tests.Cases.Inheritance.Interfaces.DefaultInterfaceMethods
+{
+	[TestCaseRequirements (TestRunCharacteristics.SupportsDefaultInterfaceMethods, "Requires support for default interface methods")]
+	class StaticDefaultInterfaceMethodOnStruct
+	{
+		public static void Main ()
+		{
+#if SUPPORTS_DEFAULT_INTERFACE_METHODS
+			Foo<Derived> ();
+#endif
+		}
+
+#if SUPPORTS_DEFAULT_INTERFACE_METHODS
+		[Kept]
+		static void Foo<T> () where T : IBase
+		{
+			T.Foo ();
```

Comment:
Nit: I would avoid `Foo`, `Bar` and similar. Policheck doesn't like those already - and it's not descriptive. Personally, I prefer descriptive names in tests - makes it easier to understand what the test does and typically makes it easier to debug.